### PR TITLE
Add stakeholder SIGs for wg-security-audit

### LINKED ISSUE #2176: update sigs.yaml with more wg info
It would be good to see if the WG is horizontal, domain-specific, etc. and maybe what SIGs they cross into on the [sigs/wg list](https://github.com/kubernetes/community/blob/master/sig-list.md). if folks are in agreement, we'd need to update sigs.yaml.

ie:

name  |  horizontal (y/n) OR list of cross cutting SIGs  |  contact  |  meeting info
---  |  ---  |  ---  |  ---

### PR DESCRIPTION
**Pull request by nzoueidi**
Add SIG Auth as the stakeholder SIGs for the Security Audit Working Group

Fixes #2176 
/kind documentation
/sig contributor-experience
/wg security-audit
/cc @nikhita @parispittman

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -2648,7 +2648,9 @@ workinggroups:
   mission_statement: >
     Perform a security audit on k8s with a vendor and produce as artifacts a threat
     model and whitepaper outlining everything found during the audit.
-
+  charter_link:
```
Comment (nikhita) [inline sigs.yaml]: Tests should pass if you keep a newline here and remove the empty `charter_link` key

Comment (nzoueidi) [reply sigs.yaml]: Thanks Nikhita, I figured out that I need to run `make` to generate the files and not edit them all manually :')

**Comment by nikhita** [comment]
/assign @cblecker 
for approval

**Pull request merged**